examples/test/chrome.mjs

```diff
@@ -1,23 +1,22 @@
-import { exit } from "process";
+import { exit } from "process"
```

Comment:
This reformat should be added to the `eslint` config to be checked inside the CI (maybe we are missing *.mjs files?)

Reply:
true, we could update package.json in a separate PR